packages/atclient/tests/noci_test_get_sharedkey.c

```diff
@@ -0,0 +1,96 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <atlogger/atlogger.h>
+#include <atclient/metadata.h>
+#include <atclient/constants.h>
+#include <atclient/atclient.h>
+#include <atclient/atkey.h>
+#include <atclient/atsign.h>
+
+#define TAG "noci_test_get_sharedkey"
+
+#define ATSIGN "@expensiveferret"
+
+#define ATKEYS_FILE_PATH "/home/realvarx/.atsign/keys/@expensiveferret_key.atKeys"
```

Comment:
Should we be merging tests like this that depend on keys locally?

Reply:
We also have examples which depend on local keys. We could merge those tests (using the noci prefix for now, so we can adjust the variables and run them locally) and convert them to ci when we setup the secrets we need for them

Reply:
I feel if that's the case, then it should live in `examples/`. It does not feel right to me to have tests that can't be ran in CI be in a `tests/` folder.

What do you think?

Reply:
It also looks like to me you were using this for manual tests while you were developing. I did something similar with my `debug.sh` script [here](https://github.com/atsign-foundation/at_c/blob/trunk/examples/desktop/crud/debug.sh)